pkg/asset/manifests/azure/cloudproviderconfig.go

```diff
@@ -0,0 +1,111 @@
+package azure
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	azureconfig "github.com/openshift/installer/pkg/asset/installconfig/azure"
+	azuretypes "github.com/openshift/installer/pkg/types/azure"
+)
+
+const (
```

Comment:
`*Default`
a) can we fall back on k8s cloudprovider to pick the defaults?
b) we have to define these defaults, can you include details in your commit message why / how these defaults were decided.

Reply:
that means bringing in the entire dependency, which is exactly what I tried to avoid, see the commit comment.
I tried and it brought 800+ files in the vendor directory. a bit overkill just to get a couple default variable?

Reply:
> that means bringing in the entire dependency, which is exactly what I tried to avoid, see the commit comment.
I tried and it brought 800+ files in the vendor directory. a bit overkill just to get a couple default variables?

hmm :confused: 

If we leave the fields corresponding to this defaults empty in the cloudconf generated here, that would kick in the defaults that cloudprovider code running in the cluster right?